hedyorg/hedy
# [FIX] level 14 comparison between float and int

[FIX] -> issue #3726 and updated tests for level 14

**Description**

Reads in all numeric values from ask as floats. Correctly compares int and floats in level 14 by having both stored as floats.

**Fixes _issue or discussion number_**
[#3726](https://github.com/hedyorg/hedy/issues/3726) 

**How to test**

Use example from bug report and confirm it returns "Above 15.0". Afterwards, change 15.0 to 15 and input 14.0 to the text form like so -

var = ask 'What is your number?'
if var > 15.0
print 'Above 15.0'
else
print 'Below'

It should output below properly when comparing an int to a float.

**Checklist**
Done? Check if you have it all in place using this list:*
  

If you're unsure about any of these, don't hesitate to ask. We're here to help!
